### PR TITLE
Bar plot support

### LINKED ISSUE #23: Using ggtips with other geoms
Hi there, I hope you guys are doing great,

First of all, thank you for developing this amazing package. I'm having issues adding reactive tooltips to a geom_bar. Here is the reproducible example that is an adaptation of your demo.

`    output$myPlot <- renderWithTooltips(
        plot = ggplot(iris, aes(x = Sepal.Width, y = Sepal.Length)) + 
geom_bar(stat = "identity",aes(fill = Species)),
        varDict = list(Sepal.Width = "Width", Sepal.Length = "Length", Species = "Species")
    )`

Please let me know if this is possible! Thank you again.

### PR DESCRIPTION
**Pull request by jcubic**
WIP: The JS part of bar plots. Waiting for Backend part. Fix #23

### CODE REVIEW

**Comment by adamgolubowski** [comment]
looks good.

File: R/grobs.R [hunk 1]
```diff
@@ -351,25 +351,25 @@ getGeomCoordsForGrob <- function(gtree,
...
     validPoints <- intersect(validX, validY)
-    
+
     geomGrob$x <- geomGrob$x[validPoints]
     geomGrob$y <- geomGrob$y[validPoints]
-    
+
     data.frame(
-      coordX = getAbsoluteX(
+      x = getAbsoluteX(
```
Comment (cosi1) [inline R/grobs.R:365]: What's the reason for renaming these columns?

File: R/misc.R [hunk 2]
```diff
@@ -72,3 +72,28 @@ getDependencies <- function() {
...
+
+
+#' Gets fill colors of bars if plot is a barplot
+#'
+#' @param p ggplot object
+#'
+#' @return vector of fill colors or NULL if no \code{rect} geom in the plot
+getBarColors <- function(p) {
+  if (!"ggplot" %in% class(p)) {
+    warning("p argument should ba a valid ggplot object")
```
Comment (cosi1) [inline R/misc.R:84]: `stop()`..?

File: R/misc.R [hunk 3]
```diff
@@ -72,3 +72,28 @@ getDependencies <- function() {
...
+    return(NULL)
+  }
+
+  gt <- gridExtra::grid.arrange(p)[[1]][[1]]
```
Comment (cosi1) [inline R/misc.R:88]: Isn't this exactly what `g` in `getTooltips()` (parent function) is?

**Pull request merged**